static/js/blockly_blocks.js

```diff
@@ -21,6 +21,32 @@ const inputBlock = {
   'tooltip': "Put the value in the final output",
 };
 
+const switchBlock = {
```

Comment:
what is the use case for this block? What games will this block help support and why have this block instead of using an integer or a free text block?

Reply:
Initially I tested my switch block on my local before integrating it with your branch and it was working fine. But I double checked that it's not working properly after merging it. So I removed this switch block.